popgen/dunn.go

```diff
@@ -73,3 +73,331 @@ func FindMinInter(g []*Group, subFa []fasta.Fasta) int {
 	}
 	return answer
 }
+
+//below is an implementation of the calculations for Tajima's D. Not currently used in the
```

Comment:
Should this be removed?  It will always be in the history of the repository, so it will not be lost.  Having it be commented out in dunn.go, might not be a place that someone will find it, unless they know about it, and would probably re-write themselves.  If you want to keep it commented out, maybe create a file in popgen that is something like "oldCode.go" which has commented out code?

Reply:
I like the idea of "oldCode" because that is more easily searchable than knowing where to go back in the codebase history to find something. There could even be a title page with the name of each function and usage statements to help people find what they are looking for.